php/utils.php

```diff
@@ -992,7 +992,9 @@ function glob_brace( $pattern, $dummy_flags = null ) {
 					}
 					$current++;
 				} else {
+					//@codingStandardsIgnoreStart
 					if ( ( '}' === $pattern[ $current ] && $depth-- === 0 ) || ( ',' === $pattern[ $current ] && 0 === $depth ) ) {
```

Comment:
It was just an oversight by me not to yodafy `$depth-- === 0` to `0 === $depth--`, there's nothing logic-wise against it.

Reply:
5f46b21c